docs/docs/running-node/configuration.md

```diff
@@ -54,6 +54,11 @@ This repository comprises four directories and seven files:
 4. `config.yaml`:
    - Contains configuration settings for the Bacalhau node.
    - Format: YAML.
+5. `update.json`:
+   - A file containing the date/time when the last version check was made.
+   - Format: JSON, e.g., `{"LastCheck":"2024-01-24T11:06:14.631816Z"}`
+6. `tokens.json`:
+   - A file containing the tokens obtained through authenticating with bacalhau clusters.
```

Comment:
Can we also link to the relevant auth section in the docs?

Reply:
Oops, was automerge.
We might consider banning that :D